backend/tests/services/test_docker_services.py

```
@@ -32,165 +25,76 @@
 
-def test_service_health_check_failure(monkeypatch):
-    """Test health check failure for services"""
 
-    with patch("subprocess.run") as mock_run:
-        # Mock ps command showing service is not running
-        mock_run.return_value = Mock(returncode=0, stdout="", stderr="")
+def test_service_health_check_http():
+    """Test health check using HTTP instead of docker commands"""
+    # This test would use HTTP calls to check service health
+    # Similar to what's in diagnostics_utils.py
+    import httpx
```

Ruff (0.12.2)

`httpx` imported but unused

Remove unused import: `httpx`

(F401)